data/json/items/tools.json

```diff
@@ -6193,6 +6193,22 @@
     }
   },
   {
+    "id": "hand_drill",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "light_gray",
+    "name": "hand drill",
+    "description": "A primitive manual drill with a single drill bit.  It is slow and it will exhaust you quickly.",
```

Comment:
@Coolthulhu said to avoid including too much stuff in this PR. As I mentioned in the description, I will create a new PR to include adding new items to the various loot tables/item groups.